app/colorCoder.js

```diff
@@ -1,99 +1,39 @@
-
-const MajorColorNames = [
-    "WHITE", "RED", "BLACK", "YELLOW", "VIOLET"
-];
-const MinorColorNames = [
-	"BLUE", "ORANGE", "GREEN", "BROWN", "SLATE"
-];
-
-function ColorPair(){
-           this.majorColor;
-           this.minorColor;
+const { MajorColorNames, MinorColorNames, ColorPair } = require("./printPairs");
+function getColorFromPairNumber(pairNumber) {
+  let minorSize = MajorColorNames.length;
+  let majorSize = MinorColorNames.length;
+  if (pairNumber < 1 || pairNumber > minorSize * majorSize) {
```

Comment:
Nice range check :)
But how would you prove that it's working? What would it take to write a test for this?